internal/middleware/valid_dev_license.go

```diff
@@ -0,0 +1,77 @@
+package middleware
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"slices"
+
+	"github.com/DIMO-Network/token-exchange-api/internal/middleware/dex"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/gofiber/fiber/v2"
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/rs/zerolog"
+	"google.golang.org/protobuf/proto"
+)
+
+// mobileAppAudience Audience in DIMO mobile JWT
+const mobileAppAudience = "dimo-driver"
+
+//go:generate mockgen -source valid_dev_license.go -destination mocks/valid_dev_license_mock.go
```

Comment:
nit: **This isn't important so you can ignore this rant of a comment***
Generally I don't think we gain much from creating mock packages for people to import. Instead we can define the go generate in the test file that needs it and give it an `_test.go` suffix. My reasoning is the current behavior creates the same type of problem when you define an interface in the implementing package. AKA If the token_exchange_test are updated to no longer need this mock we will still have this generated package hanging around unused. Secondly we are adding test specific code to our binary with little gain.
```suggestion
//go:generate mockgen -source ../middleware/valid_dev_license.go -destination mocks/valid_dev_license_mock_test.go
```
And if you made it this far thanks for reading this rant.